lib/saml/post_url_service.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module SAML
+  # This class is responsible for providing the requests for the various
+  # SSO and SLO endpoints
+  # It provides a similar interface to URLService, but for most endpoints
+  # it returns an SSO URL and form request parameters for use in a SAML POST
+  # submission, instead of a self-contained redirect URL.
```

Comment:
```suggestion
  # This class is responsible for providing the requests for the various SSO and SLO endpoints.
  # It provides a similar interface to {SAML::URLService}, but for most endpoints it returns an SSO URL and 
  # form request parameters for use in a SAML POST submission, instead of a self-contained redirect URL.
  #
  # @see SAML::URLService
  #
```

Suggestion for adding links to class references for YARD purposes:

![image](https://user-images.githubusercontent.com/11085141/85767452-a5911d80-b6dd-11ea-92b6-7aa8669cb11e.png)